src/DotNetWorker/OutputBindings/NoOutputBindingsInfo.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Functions.Worker.OutputBindings
+{
+    public class NoOutputBindingsInfo : OutputBindingsInfo
```

Comment:
Add a private constructor to this type.

Another approach is also to add a `NoOutputBindings` static property to `OutputBindingsInfo`  (you can have both).